css/index.css

```diff
@@ -97,34 +109,59 @@ button:hover {
 
 #about{
     margin-top: 50px;
+    padding: 20px;
 }
-.container{
+#experience {
+    padding: 20px;
+    background-color: #f9f9f9;
+  
+  }
+ 
+  
+  .job {
+    margin-bottom: 30px;
+  }
```

Comment:
In order to implement rows and columns, as the assignment asked, you can also use flexbox for the `job` class. Only in this case you'll use `flex-direction: column` instead of `flex-direction: row` (which is the default).

Reply:
it looks like I removed this class